packages/loader/container-loader/src/container.ts

```diff
@@ -1453,6 +1465,21 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private processRemoteMessage(message: ISequencedDocumentMessage): IProcessMessageResult {
         const local = this._clientId === message.clientId;
 
+         // Report if we're getting messages from a clientId that we previously marked inactive
+         if (!local) {
+            const client: IActiveSequencedClient | undefined =
+                this._protocolHandler?.quorum.getMember(message.clientId);
+            if (client?.isActive === false) {
+                this.logger.sendErrorEvent({
```

Comment:
I prefer fail-fast. I.e. critical error on container. The reason - container is screwed after that, it's much better to fail it (forever) but allow user to recover content via copy / paste. It's worse if we allow things to continue. This case might be not that bad, but in other cases we did not even had eventual consistency (like creating data store with same ID) - two clients continued, but refreshing one would result in loss of content. While it does not feel like we have that kind of issue here, I find it better to fail safety then take chances.

Reply:
just started a thread with matt and helio about this. i agree we should close the container. is some cases this might even prevent corruption, as it will block resubmit. it's still possible the old ops are incorrect and corrupt the document anyway